CDP4WebServices.API/Services/Headers/HeaderInfoProvider.cs

```diff
@@ -111,12 +130,27 @@ public void RegisterResponseHeaders(Response response)
         {
             response.Headers.Add(this.Cdp4ServerHeader, this.Cdp4ServerVersion);
             response.Headers.Add(this.Cdp4CommonHeader, this.Cdp4CommonVersion);
+
             if (!response.Headers.ContainsKey(this.ContentTypeHeader))
             {
                 response.Headers.Add(this.ContentTypeHeader, this.ContentTypeVersion);
             }            
         }
 
+        /// <summary>
+        /// Register the multipart CDP4 content-type header to the passed in response.
+        /// </summary>
+        /// <param name="response">
+        /// The nancy response.
+        /// </param>
+        /// <param name="boundaryString">
+        /// The boundary text in a Multipart MIME message <see href="https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html"/>
+        /// </param>
+        public void RegisterMultipartResponseContentTypeHeader(Response response, string boundaryString)
+        {
+            response.Headers.Add(this.ContentTypeHeader, $"multipart/mixed; ecss-e-tm-10-25; version=1.0.0; boundary={boundaryString}");
```

Comment:
please have a look at the the `RegisterResponseHeaders`, i think this response headers also need the following to stay inline with eachother

```
response.Headers.Add(this.Cdp4ServerHeader, this.Cdp4ServerVersion);
response.Headers.Add(this.Cdp4CommonHeader, this.Cdp4CommonVersion);
```

Reply:
That is true. 
These two headers are added in a generic method call higher up the callstack.
So no problem here.